src/lib.rs

```diff
@@ -1,27 +1,105 @@
 extern crate libc;
 use libc::{c_char, c_uchar, c_int};
+use std::ffi::CStr;
+
+pub fn message_box(title: &CStr, message: &CStr,
```

Comment:
Now that we're exposing this for public consumption, we should do the following:
* accept `&str` instead of `&CStr`, and do the conversion internally
* return `String` instead of `*const c_char`
* return `i32` instead of `c_int`

Does that make sense?